src/FlightBooking.Core/Persistence/Dynamic/IQueryableDynamicFilterExtensions.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Linq.Dynamic.Core;
+using System.Text;
+
+namespace FlightBookingApp.Core.Persistence.Dynamic
+{
+    public static class IQueryableDynamicFilterExtensions
```

Comment:
What is the purpose of this class? Do you expect complex queries that require client-side expression building?